src/js/01-nav.js

```diff
@@ -534,5 +538,16 @@
     return Array.isArray(val) ? val : [val]
   }
 
+  function isArchiveSite (navData) {
+    const t = navData.groups
+      ? navData.groups.filter((a) => hasArchiveInName(a.homeTitle) || hasArchiveInName(a.title))
+      : []
+    return t.length > 0
+  }
+
+  function hasArchiveInName (name) {
+    return String(name).toLowerCase().includes('archive')
+  }
+
```

Comment:
at the moment, I don't have a non-flaky way to tell whether it is archive site or prod site. I come up with the method to check that the group headers have 'archive' listed in their names. Not the best way, but it will do for now...

Reply:
Isn't `site-profile`  available at that time?  See https://github.com/mulesoft/docs-site-ui/blob/b5de752e22c6e8302d5ae30c90d314919a6d3819/src/partials/banner.hbs#L10

Reply:
I don't think that variable is available for the javascript. Let me do a research and see if there is a better way to do this

Reply:
I have simplified the method using `window.location.host`: [44dc7ab](https://github.com/mulesoft/docs-site-ui/pull/193/commits/44dc7ab8c702128612c9b1f03fa13c3fd439c49c). Much more reliable since it checks whether the base URL (archive.docs.mulesoft.com) contains the word 'archive' or not.